lib/rspec-console/proxy.rb

```diff
@@ -0,0 +1,21 @@
+# Proxy is really the recorder?
```

Comment:
lol i will update this comment too, leftover from when i was figuring out but didn't want to necessarily rename `Proxy` everywhere